parquet-thrift/src/main/java/org/apache/parquet/thrift/ThriftSchemaConvertVisitor.java

```diff
@@ -18,36 +18,54 @@
  */
```

Comment:
this file changed enough that you might just want to hit "view" above and read the new copy of it